include/deal.II/lac/vector_type_traits.h

```diff
@@ -0,0 +1,46 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__vector_type_traits_h
+#define dealii__vector_type_traits_h
+
+#include <deal.II/base/std_cxx11/type_traits.h>
+
+
+DEAL_II_NAMESPACE_OPEN
+
+
+/**
+ * Type trait for a serial vector, i.e. a vector class which storage is not
+ * distributed over processes.
+ * The specialization
+ *   struct is_serial_vector : std_cxx11::true_type,
```

Comment:
Actually, use
```
  @code
  struct is_serial_vector : std_cxx11::true_type
  @endcode
```
This is the doxygen way of providing a pre-formatted piece of code.